utils/task_tester/inginious-submission-anonymizer

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+
+from hashlib import sha256
+import tempfile
+import tarfile
+import shutil
+import sys
+import os
+import re
+
+#from inginious.common.filesystems.local import LocalFSProvider
+
+def anonymize(courseid:str, path: str):
+    cwd = os.getcwd()
```

Comment:
It's probably not the best practice to make code implicitly rely on the current working dir. You should rather take it as an argument, even if this argument is `.` in most cases, as other CLIs, such as `docker build` does.

Reply:
It was a temporary fix before using the FileSystemProvider whenever possible.